lessons/Early-Math/evaluating-exponents/langs/en-us/index.adoc

```diff
@@ -119,49 +139,61 @@ Ensure that students are comfortable shifting between representations before dis
 |@math{5^2}							| @math{5 \times 2}
 |@show{(coe  '(expt 5 2))}		| @show{(coe  '(* 5 2))}
 |===
+
 }
 
+@teacher{
 With a basic understanding of how exponents work, students are ready to discover how Circles of Evaluation can help them in analyzing more complicated expressions with exponents.
+}
 
 === Investigate
 
 @QandA{
-Look at the two Circles of Evaluation below:
 
 [.embedded, cols="^.^1,^.^1", grid="none", stripes="none" frame="none"]
 |===
 
 |@show{(coe  '(* 9 (expt 3 2)))}		| @show{(coe  '(expt (* 9 3) 2))}
 |===
 
-@Q{How would you translate each of these Circles of Evaluation into verbal expressions? Why are their translations different?}
+@Q{How would you translate each of these Circles of Evaluation (above) into verbal expressions? Why are their translations different?}
 @A{The Circle of Evaluation on the left is "nine multiplied by three squared," while the Circle of Evaluation on the right is "find the product of nine and three, then square it." Their translations differ because the Circles of Evaluation are organized differently.}
+
 @Q{How would you translate each of these Circles of Evaluation into mathematical expressions?}
 @A{The Circle of Evaluation on the left is @math{9 \times 3^2} while the Circle of Evaluation on the right is @math{(9 \times 3)^2}.}
 }
 
+@teacher{
```

Comment:
If it's valuable for students to recognize this, why not tell them? This seems like a note that's actually intended for students, not teachers.

I'd frame this as a @QandA or @lesson-instruction for kids to discuss on their own, then share back. Then make the main point in student-facing text.